CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file based on the
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.2.1...HEAD)
 
 ### Backward Compatibility Fixes
+- Use short array [] - so PHP 5.3 deprecated
```

Comment:
Need do it after this https://github.com/ruflin/Elastica/pull/1090#issuecomment-218279606